From 3df085e06da99123410e2394c8c6764ab0e98f3a Mon Sep 17 00:00:00 2001
From: Prabhakar Kushwaha <prabhakar.kushwaha@nxp.com>
Date: Mon, 11 Jul 2016 16:18:28 +0530
Subject: [PATCH 70/93] board: ls1012afrdm: overwrite
 CONFIG_EXTRA_ENV_SETTINGS

Kernel load address for ls1012afrdm is 0x96000000 because it has
512MB of DDR.

So overwrite CONFIG_EXTRA_ENV_SETTINGS with updated kernel_load
address.

Signed-off-by: Prabhakar Kushwaha <prabhakar.kushwaha@nxp.com>
---
 include/configs/ls1012afrdm.h |   16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/include/configs/ls1012afrdm.h b/include/configs/ls1012afrdm.h
index 3de0cea..622c774 100644
--- a/include/configs/ls1012afrdm.h
+++ b/include/configs/ls1012afrdm.h
@@ -59,4 +59,20 @@
 #define CONFIG_SYS_MEMTEST_START	0x80000000
 #define CONFIG_SYS_MEMTEST_END		0x9fffffff
 
+#undef CONFIG_EXTRA_ENV_SETTINGS
+#define CONFIG_EXTRA_ENV_SETTINGS		\
+	"initrd_high=0xffffffff\0"		\
+	"verify=no\0"				\
+	"hwconfig=fsl_ddr:bank_intlv=auto\0"	\
+	"loadaddr=0x80100000\0"			\
+	"kernel_addr=0x100000\0"		\
+	"ramdisk_addr=0x800000\0"		\
+	"ramdisk_size=0x2000000\0"		\
+	"fdt_high=0xffffffffffffffff\0"		\
+	"initrd_high=0xffffffffffffffff\0"	\
+	"kernel_start=0xa00000\0"		\
+	"kernel_load=0x96000000\0"		\
+	"kernel_size=0x2800000\0"		\
+	"console=ttyAMA0,38400n8\0"
+
 #endif /* __LS1012ARDB_H__ */
-- 
1.7.9.5

